packages/core/test/unit/logger.test.js

```diff
@@ -162,113 +157,114 @@ describe('logger', function () {
     });
 
     it('should output error message', () => {
-      var logger = logging.getLogger();
-      var expectedMessage = 'error message';
+      const logger = logging.getLogger();
+      const expectedMessage = 'error message';
       logger.error(expectedMessage);
 
-      var message = console.error.args[0][0];
-      var groups = message.match(logMessageRegex);
+      const message = console.error.args[0][0];
+      const groups = message.match(logMessageRegex);
       expect(groups[4]).to.equal('ERROR');
       expect(groups[5].trim()).to.equal(expectedMessage);
     });
 
     it('should not output if message and meta falsey', () => {
-      var logger = logging.getLogger();
+      const logger = logging.getLogger();
       logger.error();
       logger.error(null);
       logger.error('', null);
 
-      spies.forEach(spy => expect(spy).not.to.be.called);
+      spies.forEach(spy => sinon.assert.notCalled(spy));
```

Comment:
Just for my knowledge, why replace some of these `expect`s with sinon assertions?